sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.7.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="early unlocking of encrypted systems via ssh for dracut"
+HOMEPAGE="https://github.com/dracut-crypt-ssh/dracut-crypt-ssh"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-kernel/dracut"
+RDEPEND="${DEPEND}
+	net-misc/dhcp
```

Comment:
Just out of curioisity: Would any other dhcp client work?

Reply:
It should, I don't see why something like e.g. dhcpd would not work

Reply:
... because then you have to do something "ugly" and make it depend at-least-one-of them. Too bad we don't have *virtual/dhcp* package, you'll have to do something like https://gitweb.gentoo.org/repo/gentoo.git/tree/virtual/mta/mta-1-r1.ebuild

Reply:
yikes!